### ISSUE TITLE
Tradução de lição: "Criação de Redes Neurais Convolucionais Profundas Para Clasificação de Imagens"

### ISSUE THREAD

**Issue by DanielAlvesLABDH** [programminghistorian/ph-submissions#576]
O Programming Historian em Português recebeu a proposta de tradução da lição [Creating Deep Convolutional Neural Networks for Image Classification](https://programminghistorian.org/en/lessons/image-classification-neural-networks). Agradeço ao @grut-br pela proposta. Já fiz uma primeira revisão da mesma. @anisa-hawes irá fazer as verificações necessárias. Irei convidar um editor para acompanhar a revisão desta tradução.
A @saletefarias ficará como uma das revisoras, uma vez que esta tradução resultou de um trabalho de um dos seus alunos. Agradeço a colaboração da Salete e da sua turma do Instituto Federal de Educação Tecnológica do Maranhão.

**Comment by DanielAlvesLABDH** [comment]
A @daniellesanchesDH aceitou ficar como editora desta proposta de tradução. Ela conduzirá o processo e irá fazer os convites para a segunda revisão. Muito obrigado e bom regresso!

**Comment by anisa-hawes** [comment]
Obrigada, @DanielAlvesLABDH.

Pode encontrar os ficheiros principais aqui:

- .md: [/pt/esbocos/traducoes/redes-neurais-classificacao](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/pt/esbocos/traducoes/redes-neurais-classificacao.md)
- images: [/images/image-classification-neural-networks](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/images/image-classification-neural-networks)
- assets: [/assets/image-classification-neural-networks](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/image-classification-neural-networks)

E pode consultar a pré-visualização da lição aqui:

- <http://programminghistorian.github.io/ph-submissions/pt/esbocos/traducoes/redes-neurais-classificacao>

**Comment by DanielAlvesLABDH** [comment]
Thanks a lot @anisa-hawes!

**Comment by saletefarias** [comment]
Olá Daniel, os meus alunos acharam super interessante o projeto PH. Espero no futuro, com outras turmas, continuar a apoiar o projeto. Estou de férias este mês mas assim que retornar para as demandas diárias inicio o processo de revisão. Abraços.

**Comment by DanielAlvesLABDH** [comment]
Olá @daniellesanchesDH, tudo bom? Como está o processo de convite de revisores? Obrigado

**Comment by ericbrasiln** [comment]
Olá @sanchesdani 

Substituí o Daniel na função de editor-chefe da revista em português. 
Estou dando uma passada geral nas issues das nossas traduções e tentando entender em que pé elas estão 😄

Temos alguma novidade aqui?
Abraços

**Comment by sanchesdani** [comment]
Olá Eric,

Bom dia! Obrigada pela sua consulta! Infelizmente não temos novidades...
vou tentar retomar o convite para revisores nesse segundo semestre.

Abraços,

Danielle Sanches.

Em seg., 22 de abr. de 2024 às 22:21, Eric Brasil ***@***.***>
escreveu:

**Comment by ericbrasiln** [comment]
@sanchesdani @daniellesanchesDH 
Como vai?

Alguma novidade para essa lição?

**Comment by ericbrasiln** [comment]
@daniellesanchesDH @sanchesdani 

Oi Danielle, 

O Alisson Soares @SoaresAlisson aceitou ser revisor dessa lição! Podemos avançar com ela e como é uma tradução, podemos manter apenas uma revisor, além da leitura da editora. O que acha?

**Comment by sanchesdani** [comment]
Ah, excelente!!





Em seg., 11 de nov. de 2024 às 16:01, Eric Brasil ***@***.***>
escreveu:

**Comment by ericbrasiln** [comment]
Oi @SoaresAlisson 

aqui você pode envontrar os arquivos referentes à lição. Pode colocar a revisão em comentário aqui na _issue_ e usar tasklist (`- [ ] `) colocando a linha de referência do markdown.

* .md: [/pt/esbocos/traducoes/redes-neurais-classificacao](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/pt/esbocos/traducoes/redes-neurais-classificacao.md)
* images: [/images/image-classification-neural-networks](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/images/image-classification-neural-networks)
* assets: [/assets/image-classification-neural-networks](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/image-classification-neural-networks)

E pode consultar a pré-visualização da lição aqui:

* http://programminghistorian.github.io/ph-submissions/pt/esbocos/traducoes/redes-neurais-classificacao

Qualquer dúvida pode marcar a @sanchesdani ou à mim.

Mais uma vez obrigado e abraços

**Comment by charlottejmc** [comment]
Olá @SoaresAlisson, how are you?

We are still waiting for your review of @grut-br's translation. Are you still able to commit to providing your feedback, ideally over the next few weeks?

As Eric outlined in [his comment above](https://github.com/programminghistorian/ph-submissions/issues/576#issuecomment-2483020039):

Podem incluir suas revisões e sugestões em um comentário aqui na _issue_, de preferência utilizando tasklist (`- [ ]`), e indicando a linha do markdown.

Os arquivos referentes à lição podem ser acessados nos seguintes links:

* .md: [/pt/esbocos/traducoes/redes-neurais-classificacao](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/pt/esbocos/traducoes/redes-neurais-classificacao.md)
* images: [/images/image-classification-neural-networks](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/images/image-classification-neural-networks)
* assets: [/assets/image-classification-neural-networks](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/image-classification-neural-networks)
* pré-visualização da lição: [http://programminghistorian.github.io/ph-submissions/pt/esbocos/traducoes/redes-neurais-classificacao](http://programminghistorian.github.io/ph-submissions/pt/esbocos/traducoes/redes-neurais-classificacao)

Coloco aqui as [Diretrizes para revisores](https://programminghistorian.org/pt/directrizes-revisor).

If you have any questions, please let us know.

**Comment by SoaresAlisson** [comment]
(para não atrasar mais, envio a revisão da tradução e esta semana testo os códigos)

## Observações gerais:
Não tenho muita experiência com textos técnicos em português de Portugal, então minhas observações tendem para o padrão português brasileiro. Uma questão se refere a termos em inglês, se devem ser traduzidos, se devem permanecer em inglês ou ainda, uma solução híbrida, com ambas opções. No português brasileiro, deixamos muitos dos termos em inglês no original. Um argumento a favor de possuir os termos originais em inglês e traduzido é tornar o texto mais "encontrável" em mecanismos de busca. Na linha 134, por exemplo, há o uso tanto do termo em inglês como do português. Talvez seja interessante usar este mesmo padrão em todo o texto, para termos chave como "Transformers", "Convolutional Neural Networks", etc. O que acham?

## Sobre a notação adotada:
Destaco duas alterações na notação adotada para mostrar as minhas sugestões.
1) Ao longo da checagem, eu acabei montando um script (em R) para intercalar as frases e facilitar a leitura. Assim, algumas sugestões minhas contêm o número da página no Markdown e a partir da linha 52 possuem também qual o número da frase no parágrafo (linha do Markdown). Por exemplo,, 52.1 significa linha 52, frase 1; 54.2 significa linha 54, frase 2. 
2) Outras sugestões de alteração acrescida foi de utilizar marcação Markdown, para num mesmo trecho de frase indicar o que seria retirado e a sugestão de substituição. O texto tachado indica o trecho a ser retirado e o negrito a parte a ser acrescida.

## Revisão / Sugestões
- [ ] 38. "Yet, the high level of statistics and computer science knowledge required to comprehend machine learning algorithm", foi traduzido como: "No entanto, o alto nível de estatística e conhecimento de ciência da computação necessários...", mas eu sugeriria: "No entanto, **os altos níveis** de estatística e conhecimento de ciência da computação necessários..."
- [ ] 40 "que, juntamente com os transformadores, são modelos de aprendizado de máquina". Posso sugerir acrescentar "que, juntamente com os transformadores (__transformers__), são modelos de aprendizado de 
- [ ] 52.1. "Along with JavaScript", ao invés de "Junto com o JavaScript", utilizar algo como "Além do JavaScript".
- [ ] 46.3, 48.2, 54.1 e 70.2 traduzir "coding" como "programação" ao invés de "codificação"?
- [ ] 64.2. "You can download a .zip file", foi traduzido como ~"Você pode \[baixar um.ziparquivo contendo"~, mas 1) seria melhor "Você pode \[baixar um arquivo .zip contendo"; 2) "file" foi traduzido aqui como "arquivo", mas em outras frases apareceu como "ficheiro".
- [ ] 78.2. Sugestão: "O seguinte é um diagrama simplificado[^3] de como ~funcionam um neurônio biológico e artificial~ **um neurônio biológico e um neurônio artificial funcionam**." 
- [ ] 84.6. Sugestão: "Para ~combater~ **evitar** isso, os neurônios artificiais determinam se a soma das entradas que recebem" 
- [ ] 102.7  "No entanto, sabemos que precisamos obter uma resposta de 7,5, então uma das coisas que podemos fazer é multiplicar ~o~ **as** entradas por um número.
- [ ] 102.12 "Vamos tentar 1.5: ~(3x1.5)+(2x1.5)=7.5~ `(3 x 1,5) + (2 x 1,5) = 7,5`." Usar vírgula no lugar do ponto como separador de decimais e espaço entre números para facilitar a leitura.
- [ ] 116.2 "~Basicamente~ **Em sua forma mais básica**, uma convolução é uma..." 
- [ ] 116.3 - "Se você **já** usou filtros, como desfoques"
- [ ] 120.1 - ~"Finalmente, uma “camada de pooling” é utilizada"~ alterar para 'Finalmente, uma “camada de pooling” (agrupamento) é utilizada.'. Pooling aqui apareceu sem tradução, mas na seguinte apareceu traduzida, o que pode confundir o leitor.
- [ ] 120.2 - ~pois usa uma grade; geralmente 2x2 e o passa~ . Sugestões: "pois usa uma grade, geralmente 2x2, e o passa sobre" ou "pois usa uma grade - geralmente 2x2 - e o passa sobre"
- [ ] 130.2 - "At first, these features" > "~A princípio~ **Em um primeiro momento**, esses recursos são mapeados para".
- [ ] 130.3 "&mdashdetectar recursos e formas básicas&mdash" &mdash apareceu como texto puro.
- [ ] 136.2 - "Você descobrirá que pode *“Upload”*" > "Você descobrirá que pode **fazer o** *“Upload” **(tradução: carregar)** de imagens ou" ou "Você descobrirá que pode **fazer o** *“Upload” **(carregar)** de imagens ou"
- [ ] 138.2 - "Selecione a pasta “aircraft” (em inglês) (tradução: ~eoronaves~ **aeoronaves**) de dentro da pasta “training” (em inglês) **(tradução: treinamento)** do conjunto de dados e arraste-a para a janela Teachable Machine."
- [ ] 144.1 - ~'Depois de terminar de carregar as imagens, você pode ajustar vários parâmetros de como o modelo deve ser treinado clicando em “Advanced” (em inglês) (tradução: Avançado) em "Training" (em inglês) (tradução: Treinamento'~ . Sugestão: '**Após** terminar de carregar as imagens, você pode ajustar vários parâmetros de como o modelo deve ser treinado, **dentro de** "Training" (em inglês) (tradução: Treinamento) clicando em “Advanced” (em inglês) (tradução: Avançado).'
).
- [ ] 144.2 - Original: "Aqui você verá opções para épocas, tamanho do lote e taxa de aprendizado." sugestão de alteração: "Aqui você verá opções para "epochs" (tradução: épocas), "batch size" (tradução: tamanho do lote) e "learning rate" (tradução: taxa de aprendizado)."
- [ ] 170.5 - "[Ml5.js](https://perma.cc/3GBE-NVER) e [p5.js](https://perma.cc/5DXT-UR72) , que usaremos posteriormente para incorporar nosso modelo em nosso *site*, dependem do Tensorflow.js em um nível inferior." > "[Ml5.js](https://perma.cc/3GBE-NVER) e [p5.js](https://perma.cc/5DXT-UR72), que usaremos posteriormente para incorporar nosso modelo em nosso *site*, dependem do Tensorflow.js em um nível inferior." (retirada de espaço antes da vírgula)
- [ ] 186.1 - "[Ml5.js](https://ml5js.org/) é uma biblioteca JavaScript construída sobre **o** Tensorflow.js."
- [ ] 257.4 - "Basicamente, o JavaScript lê o código de cima para baixo, mas não ~para~ **interrompe** para concluir nenhuma parte do código, a menos que seja necessário.". Tradução correta, porém para evitar "para para", sugiro outro termo equivalente, como interromper, pausar, etc.
- [ ] 303. "Essa configuração permanecerá ~em vigor~ **neste local** até que você decida alterá-la." 
- [ ] 327.3 - "**Isto requer** ~Requer~ um único argumento"

**Comment by SoaresAlisson** [comment]
Hello @charlottejmc, 

I am reviewing [Criação de Redes Neurais Convolucionais Profundas Para Clasificação de Imagens](http://programminghistorian.github.io/ph-submissions/pt/esbocos/traducoes/redes-neurais-classificacao), but the [links on this comment](https://github.com/programminghistorian/ph-submissions/issues/576#issuecomment-2651182103) are from another lesson, [Visualização básica de dados tabulares com R](http://programminghistorian.github.io/ph-submissions/pt/esbocos/originais/visualizacao-basica-dados-tabulares-r). It was supposed to review both lessons?

**Comment by charlottejmc** [comment]
Hi @SoaresAlisson, thank you very much for your review of  [Criação de Redes Neurais Convolucionais Profundas Para Clasificação de Imagens](http://programminghistorian.github.io/ph-submissions/pt/esbocos/traducoes/redes-neurais-classificacao). You are right, the links in my comment were wrong, I'm very sorry for the confusion! We do not need you to review the other lesson [Visualização básica de dados tabulares com R](http://programminghistorian.github.io/ph-submissions/pt/esbocos/originais/visualizacao-basica-dados-tabulares-r).

**Comment by anisa-hawes** [comment]
Olá Pedro Lucas @grut-br,

I hope you are well? Our Managing Editor Eric @ericbrasiln [has suggested](https://github.com/programminghistorian/ph-submissions/issues/576#issuecomment-2468842944) that, in order to expedite publication after a long delay, we try to move forwards with your translation following one peer review only. 

Do you have scope to work through [Alisson's feedback](https://github.com/programminghistorian/ph-submissions/issues/576#issuecomment-2667125681) and make some revisions to your translation?

I've granted you write access to our repository, so that you can make direct edits to your file: [/pt/esbocos/traducoes/redes-neurais-classificacao.md](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/pt/esbocos/traducoes/redes-neurais-classificacao.md).

If you are able to set aside time to work on this within the next 30 days, we can move towards publication as soon as possible.
What timeline would be manageable for you?

With our best wishes,
Anisa and Charlotte

**Comment by anisa-hawes** [comment]
Olá Pedro Lucas @grut-br, 

I'm writing to follow up on my [previous note](https://github.com/programminghistorian/ph-submissions/issues/576#issuecomment-2684736310). I'd like to ask if you have any capacity to work through [Alisson's feedback](https://github.com/programminghistorian/ph-submissions/issues/576#issuecomment-2667125681) and make some revisions to your translation?

I realise that there has been several extended delays in the development of your translation, so we do understand if you don't feel able to revisit this work at the moment. But if you are interested in working through the suggested revisions, then we will be pleased to collaborate.

With best wishes,
Anisa

**Comment by anisa-hawes** [comment]
Dear Pedro Lucas @grut-br,

We are closing this Issue due to inactivity. Please write to our Managing Editor @ericbrasiln if you decide that you'd like to return to work on this translation later in 2025.

With very best wishes,
Anisa
